sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -298,28 +288,28 @@ def enqueued_time(self):
         return None
 
     @property
-    def partition_key(self):
-        # type: () -> Optional[bytes]
+    def partition_key(self) -> Optional[bytes]:
         """The partition key of the event.
 
         :rtype: bytes
         """
-        try:
-            return self._raw_amqp_message.annotations[PROP_PARTITION_KEY_AMQP_SYMBOL]
-        except KeyError:
-            return self._raw_amqp_message.annotations.get(PROP_PARTITION_KEY, None)
+        # TODO: Ask Anna. can we remove the try and just do except? Haven't seen a case where symbol is used to get.
+        # try:
+        #    return self._raw_amqp_message.annotations[types.AMQPSymbol(PROP_PARTITION_KEY)]
+        # except KeyError:
+        #    return self._raw_amqp_message.annotations.get(PROP_PARTITION_KEY, None)
+        return self._raw_amqp_message.annotations.get(PROP_PARTITION_KEY, None)
```

Comment:
question @annatisch : can we just do the following?

Reply:
Since this partition key is only added to the message.annotations/raw_amqp_message.annotations in the `send` method and is not settable on the EventData, I don't think this would ever return anything other than None before sending if the user calls it. So keeping only the except logic here, unless there are any concerns?